server/src/main/java/org/eclipse/hono/server/HonoServer.java

```diff
@@ -86,7 +81,7 @@
      */
     @Autowired(required = false)
     public HonoServer setHonoConfiguration(final HonoConfigProperties props) {
```

Comment:
I think it makes sense to let the concrete implementing class handle specific code and thus left is here.
In the HonoServer class this currently only consists of the warning - but this warning is completely specific to Hono server and will not fit with protocol adapters e.g.
So the method is called via super, and some parts remain here.